classes/monograph/PublishedMonographDAO.inc.php

```diff
@@ -17,6 +17,13 @@
 import('classes.monograph.PublishedMonograph');
 import('classes.monograph.MonographDAO');
 
+define('ORDERBY_DATE_PUBLISHED_ASC', 'datePublishedASC');
+define('ORDERBY_DATE_PUBLISHED_DESC', 'datePublishedDESC');
+define('ORDERBY_TITLE_ASC', 'titleASC');
+define('ORDERBY_TITLE_DESC', 'titleDESC');
+define('ORDERBY_SERIES_POSITION_ASC', 'seriesPositionASC');
+define('ORDERBY_SERIES_POSITION_DESC', 'seriesPositionDESC');
```

Comment:
I don't mind if it's a single drop-down on the front end, it just seems awkward to have the DAOs receiving both direction-and-field and direction.